codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/config/ServiceConfigGenerator.kt

```diff
@@ -78,7 +78,6 @@ sealed class ServiceConfig(name: String) : Section(name) {
     object BuilderBuild : ServiceConfig("BuilderBuild")
 }
 
-// TODO: if this becomes hot, it may need to be cached in a knowledge index
```

Comment:
What is a knowledge index? I'm not familiar with that term

Reply:
It's a smithy thing for caching information gleaned from the model